galsim/zernike.py

```diff
@@ -499,6 +504,17 @@ def __init__(self, coef, R_outer=1.0, R_inner=0.0):
         self.R_outer = float(R_outer)
         self.R_inner = float(R_inner)
 
+    @staticmethod
```

Comment:
Factory methods are more typically classmethod.  I don't think there is much functional difference between them, at least in the absence of a class hierarchy.  But I believe it's more pythonic to use a classmethod here.  (And below for _from_uvxy)